### PR TITLE
setup.py: Makes setting.json pretty

### PR DESCRIPTION
**Pull request by DanRadulescu962**
I added a newline after each JSON object.

Fixes https://github.com/NIT-dgp/UIP/issues/158

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```

**Comment by DanRadulescu962** [comment]
I think I have the right structure.

**Comment by abhsag24** [comment]
@DanRadulescu962 yeah that's a generic message :P that gitmate comments on every PR

File: setup.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def get_requirements(filename):
                              'https://www.reddit.com/r/pics/',
                              'https://api.desktoppr.co/1/wallpapers', ]}
     with open(settings_file_path, "w") as settings_file:
-        settings_file.write(json.dumps(file_data))
+        settings_file.write(json.dumps(file_data) + '\n')
```
Comment (abhsag24) [inline setup.py]: hey so can you look at `update_settings` in `utils` ? i think that has the right way this just adds a a newline at then end of the json

**Comment by abhsag24** [comment]
unack d7cb880

File: setup.py (hunk 1 above)
Comment (DanRadulescu962) [inline setup.py]: I only have __init__.py  setupUtils.py  utils.py

**Comment by abhsag24** [comment]
ack 7dec6c7

**Comment by nemaniarjun** [comment]
ack 7dec6c7

**Comment by abhsag24** [comment]
cmon gitmate :P

**Comment by abhsag24** [comment]
ack 7dec6c7

**Comment by abhsag24** [comment]
finally! XD

**Pull request merged**